tests/rpm/source_date_epoch/BUILD

```diff
@@ -12,8 +12,8 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-load("//:mappings.bzl", "pkg_filegroup", "pkg_files")
-load("//:rpm.bzl", "pkg_rpm")
+load("@rules_pkg//pkg:mappings.bzl", "pkg_filegroup", "pkg_files")
+load("@rules_pkg//pkg:rpm.bzl", "pkg_rpm")
```

Comment:
Is the repo prefix really truly necessary here?

Reply:
No. I guess not.  But.... while we are looking.
Is there any way you can make this test conditional? 
It does not run in CI (no rpmbuild) and it fails locally for me (wrong rpmbuild version).

That would be a pain, becuase the rpmbuild version might have to be part of the auto-detected toolchain.

Reply:
Huh.  I thought that they would be skipped, given what I have here:

https://github.com/bazelbuild/rules_pkg/blob/main/pkg/tests/rpm/source_date_epoch/rpm_contents_vs_manifest_test.py#L57-L64

That change may simply not be working, though.

Reply:
Tracking this in https://github.com/bazelbuild/rules_pkg/issues/455